packages/suite-desktop/src-electron/processes/BaseProcess.ts

```diff
@@ -37,12 +37,7 @@ class BaseProcess {
      * - service: The service is working
      * - process: The process is running
      */
-    async status() {
-        return {
-            service: false,
-            process: false,
-        };
-    }
+    abstract status(): Promise<{ service: boolean; process: boolean }>;
```

Comment:
this is one "more than trivial" change, so if you could check please @keraf 
I'd say that using abstract class here is fine (If I understand correctly that BaseClass shall never be used directly anywhere)